.github/workflows/ci.yml

```diff
@@ -15,6 +26,37 @@ jobs:
       - name: "Build binaries"
         run: make build
 
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.x
+
+      - name: "Generate Heimdall Params"
+        run: go run helper/heimdall-params.template.go $(network)
+
+      - name: "Run linter"
```

Comment:
It might be a bit dangerous to run all linters against legacy code. In Bor I preferred to run linters only against changed files https://github.com/maticnetwork/bor/blob/develop/.golangci.yml#L187

Reply:
Well, you may change one file and affect another one. Could you clarify what could be dangerous?

Reply:
Oh, can I just take that config file from bor and use here?

Reply:
Done